apps/coordinator/ts/common/chain.ts

```diff
@@ -4,23 +4,23 @@ import { ErrorCodes } from "./errors";
 import { ESupportedNetworks } from "./networks";
 
 /**
- * Get the RPCUrl for Alchemy based on the chain we need to interact with
+ * Get the RPC url for the chain we need to interact with
  *
  * @param network - the network we want to interact with
- * @returns the RPCUrl for the network
+ * @returns the RPC url for the network
  */
-export const getAlchemyRpcUrl = (network: ESupportedNetworks): string => {
-  const apiKey = process.env.RPC_API_KEY;
+export const getRpcUrl = (network: ESupportedNetworks): string => {
+  const { COORDINATOR_RPC_URL } = process.env;
```

Comment:
You should use it like this because sometimes env values are empty when you use destructuring.

```suggestion
  const rpcUrl  = process.env.COORDINATOR_RPC_URL;
```